### ISSUE TITLE
About how to train it

### ISSUE THREAD

**Issue by bull-server** [brijeshiitg/Pytorch-implementation-of-SRNet#2]
I download it and train it with my train profile, but it cannot converge, So I want consult on how to train it. Aprreciate for any answer.

**Comment by mr3543** [comment]
Do you know the data normalization scheme used for the network? i.e. images /= 255 or mean/std normalization. thanks

**Comment by quantum-soul** [comment]
hey, friend, have you solved the problem yet? If it is solved, please share the solution. Thank you

**Comment by zzb112233** [comment]
> hey, friend, have you solved the problem yet? If it is solved, please share the solution. Thank you

Hi,Have you got it？waiting for reply...

**Comment by brijeshiitg** [comment]
Write your own training code with data augmentation.
